test/InterversionTests/MixedClusterTests.cs

```diff
@@ -479,23 +646,23 @@ public async Task SubscriptionsInMixedCluster_FailoverFrom410To41()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "WIP")]
         public async Task V40Cluster_V41Client_BasicReplication()
         {
-            (var urlA, var serverA) = await GetServerAsync("4.0.7");
-            (var urlB, var serverB) = await GetServerAsync("4.0.7");
-            (var urlc, var serverC) = await GetServerAsync("4.0.7");
+            var nodeA = await GetServerAsync("4.0.7");
```

Comment:
In places like this, please make sure we kill the process in the end of the test that uses it. I know it's a broader issue, but It will prevent long debug sessions looking for reasons to tests slowdown etc.

Reply:
Those are skipped, haven't addressed them at all.

Reply:
@karmeli87 right